.github/dependabot.yml

```diff
@@ -11,8 +11,6 @@ updates:
       interval: 'daily'
     ignore:
       - dependency-name: '@hmpps-book-secure-move-frameworks*'
-      - dependency-name: 'govuk-frontend'
```

Comment:
@thomasleese be interesting to get your thoughts on this. Initially this was put in to prevent potential major visual changes. But I think its just lead to us not staying more current.

And also looking at the way GOV.UK Frontend is released, they provide opt-in options for major visual changes, for example the new link styles. 

I personally would prefer us to be notified of these releases, but maybe they come with a deeper review of the automated PRs?

Reply:
> I personally would prefer us to be notified of these releases, but maybe they come with a deeper review of the automated PRs?

Yeah, I completely agree with this. It's not like we're overwhelmed at the moment with the number of PRs, so I think it's reasonable to reduce the number that we're ignoring. Plus regardless of what is being upgraded, we should spend time reviewing them as appropriate, something which upgrades a small library we use in the tests require less in depth analyse as a major update or `govuk-frontend` for example.

Going on a bit of a tangent, have we ever thought about putting in place some visual regression testing?

Reply:
>Going on a bit of a tangent, have we ever thought about putting in place some visual regression testing?

Yeah, it has been something we've discussed but the main reason had just been capacity previously. It's maybe something that would be good to include moving forward when we hand over into a maintenance state.

On a related note I spotted that images are no longer showing, I'm guessing after a recent browserify upgrade, and this type of thing would have caught it.

I've used [wraith](https://github.com/bbc/wraith) in the past but not for a little while. Are there any libraries you've had a good experience with?

Reply:
> I've used [wraith](https://github.com/bbc/wraith) in the past but not for a little while. Are there any libraries you've had a good experience with?

Wraith is the only library I've used in the past too, but I understand that it's not really updated any more.

Reply:
Yep, unfortunately so. Last update was 3 years ago...